wire-ios/Wire-iOS/Wire-Info.plist

```diff
@@ -154,8 +152,15 @@
 	</array>
 	<key>UIFileSharingEnabled</key>
 	<true/>
-	<key>UILaunchStoryboardName</key>
-	<string>LaunchScreen</string>
+	<key>UILaunchScreen</key>
+	<dict>
+		<key>UIColorName</key>
+		<string>Black</string>
+		<key>UIImageName</key>
+		<string>Wire/Shield</string>
+		<key>UIImageRespectsSafeAreaInsets</key>
+		<true/>
+	</dict>
```

Comment:
I didn't know it's possible, so it just shows a black screen with a centred image?
https://developer.apple.com/documentation/bundleresources/information_property_list/uilaunchscreen